### DISCUSSION TITLE
Car shows travel to CA!!!!!

### DISCUSSION THREAD

**Discussion by cleanev** [teslamate-org/teslamate#3028] [Q&A]
Car got an update to 2022.44.25.2 this morning however when checking ‘Visited’ dashboard my car shows driven to CA!!!!!
Narrowed down to 23h to 21h from now (4:30EST). However I have never driven to California. How to remove this extraneous entry? And why does this show up? Snapshots show this anomaly while two additional screenshots show there seems to be no corresponding entry for drives in last 24hours.

Is this a bug of some sort that needs to be flagged under Issues?

Your assistance welcome. Happy holidays all.

![93676685-3E29-427C-B7E5-0AEB0005C4B4](https://user-images.githubusercontent.com/45519661/208770728-5c140b85-3870-494c-aa4c-6dca274fa199.png)
![9F23F80F-A11D-4418-B249-EAFAF5C74C1A](https://user-images.githubusercontent.com/45519661/208770897-1c22b22b-839e-4d9b-ba0c-9ce5d672efc9.jpeg)
![9760F8FF-9919-400E-80B7-7657FF1EDF52](https://user-images.githubusercontent.com/45519661/208770901-fd44bbc1-ca58-4e06-89a3-0da26061dd9e.jpeg)

**Comment by cwanja** [comment]
It is not a TeslaMate bug. Your car pinged the mothership and it showed the GPS that it was there. If anything, it's a Tesla bug. But you can just go drop the positions from the `positions` table.

**Comment by cleanev** [comment]
Hi @cwanja - thanks for quick turnaround. would you happen to know how to remove this entry from positions table that you mention above? I am not sure how to go about taking that action. 
Appreciate it.

**Reply by cwanja** [reply]
Of course @cleanev. I am not at my computer to give you exact lines of code, but these two threads should help #1550 #2699

Please respond here after reviewing those where you are stuck and I can respond in a few hours to help.

**Reply by cleanev** [reply]
I followed guidance in [https://github.com/adriankumpf/teslamate/issues/1550](url) and got rid of 24 rows! Wonder how many times it ping in very short timeframe.

Thank you so much @cwanja.

Happy holidays and have a safe and healthy time.

**Reply by cwanja** [reply]
Awesome! Glad to hear that it's fixed. 

Happy holidays to you as well.
